mitmproxy/addons/__init__.py

```diff
@@ -1,6 +1,7 @@
 from mitmproxy.addons import anticache
 from mitmproxy.addons import anticomp
 from mitmproxy.addons import block
+from mitmproxy.addons import blockech
```

Comment:
`blockech` does not read super well - maybe `block_ech` for the file and option name?